x/ref/envvar.go

```diff
@@ -22,6 +22,10 @@ const (
 	// See v.io/x/ref/lib/security.CreatePersistentPrincipal.
 	EnvCredentials = "V23_CREDENTIALS"
 
+	// When set and non-empty, EnvCredentials is hosted on a read-only
+	// filesystem.
+	EnvCredentialsReadonlyFileSystem = "V23_CREDENTIALS_EADONLY_FILESYSTEM"
```

Comment:
Typo - `V23_CREDENTIALS_READONLY_FILESYSTEM`

I think we can live with out this flag - IMO the choice is use the previous `V23_CREDENTIALS_NO_LOCK` flag so our code does not change - or we go with the cleaner model. Adding a new environment variable seems lose/lose.